app/models/concerns/taggable.rb

```diff
@@ -98,10 +98,12 @@ def tag_string=(tag_string)
       string.strip!
       next if string.blank?
 
-      self.tags << (Tag.find_by_name(string) || UnsortedTag.create(name: string))
+      # AO3-6145: tags list has duplicates that need to be removed or
```

Comment:
I think we can remove this comment. Usually we don't need to reference specific issue IDs in comments (unless it's a TODO or a temporary fix we plan to revisit at some point).